tests/functional/raw-no-data-bad-column/source/data/config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parameters": {
+    "db": {
+      "endpoint": "%env(string:ENDPOINT)%",
+      "#key": "%env(string:KEY)%",
+      "databaseId": "%env(string:DATABASE_ID)%"
+    },
+    "containerId": "simple",
+    "query": "SELECT * FROM c ORDER BY c.foo.bar.invalid",
```

Comment:
Kluc `c.foo.bar.invalid` sa nenachadza v ziadnom dokumente v kontajnery `simple` ... ale container `simple` nie je prazdny (edge case ked to padalo).